### DISCUSSION TITLE
Webview2 - Is it supported on net8.0-desktop

### DISCUSSION THREAD

**Discussion by rruprai1** [unoplatform/uno#17990] [Q&A] [answered]
Is webview2 support on net8.0-desktop for MacOS? It seems Linux is not supported.

Docs suggest it is: https://platform.uno/docs/articles/controls/WebView.html

however:
warning Uno0001: Microsoft.UI.Xaml.Controls.WebView2.CoreWebView2 is not implemented in Uno (https://aka.platform.uno/notimplemented?m=Microsoft.UI.Xaml.Controls.WebView2.CoreWebView2) (https://aka.platform.uno/notimplemented) [/....csproj::TargetFramework=net8.0-desktop]

Using uno.sdk 5.2.132.

Could I be missing something in csproj to include? Right now I only have Hosting and ThemeService

**Comment by morning4coffe-dev** [comment, marked as answer]
Hey @rruprai1,

Thank you for your question. I believe that WebView2 is currently supported only on macOS Catalyst (`net8.0-maccatalyst`), not on macOS Skia (`net8.0-desktop`). You can refer to this discussion for more details: [GitHub Discussion Link](https://github.com/unoplatform/uno/discussions/14709). Could you please test this out when you build for this target?

CC @MartinZikmund We should clarify this in the documentation, as it may be confusing.

**Reply by MartinZikmund** [reply]
@morning4coffe-dev you are right, feel free to make PR for the docs update please

**Reply by rruprai1** [reply]
I think there is till an issue, I've compiling using net8.0-maccatalyst, following the example on uno docs for webview2, https://platform.uno/docs/articles/controls/WebView.html and still not seeing the content load.

Using Uno.SDK 5.3.99

**Reply by rruprai1** [reply]
Actually NVM, I have messed up lauch/task settings.

**Comment by rruprai1** [comment]
Thanks for that info, I didn't know about net8.0-macos, is there any doc about it on UNO?

Right now I've been using the online docs and following:
https://platform.uno/docs/articles/getting-started/requirements.html

**Reply by morning4coffe-dev** [reply]
@rruprai1 My apologies, the correct target is net8.0-maccatalyst. The documentation you are following should be up-to-date, and if you use our wizard with Visual Studio or on [new.platform.uno](https://new.platform.uno/), you should be able to select this target without any issues:
![image](https://github.com/user-attachments/assets/3cf4fd1d-d86e-450d-a632-048875aeac46)
Please let me know if this helps.

**Reply by rruprai1** [reply]
No worries. Thanks.
